cmd/puller/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"flag"
+	"github.com/kubeflow/kfserving/pkg/puller"
+	"log"
+)
+
+var (
+	modelDir = flag.String("model_dir", "/tmp","directory for multi-model config files")
+)
+
+func main() {
+	flag.Parse()
+	puller.OnConfigChange(func(e puller.EventWrapper) {
+		log.Println("Send a request to:", e.LoadState)
```

Comment:
This section will do a load / unload call.

EventWrapper will have information pertaining to:
1. Which model
2. Whether load / unload
3. What information should it be a load